include/unifex/thread_unsafe_event_loop.hpp

```diff
@@ -149,6 +149,16 @@ namespace _thread_unsafe_event_loop {
       return after_operation<Duration, std::remove_cvref_t<Receiver>>{
           (Receiver &&) r, duration_, loop_};
     }
+    template <typename Receiver>
+    after_operation<Duration, std::remove_cvref_t<Receiver>> connect(Receiver&& r) & {
+      return after_operation<Duration, std::remove_cvref_t<Receiver>>{
+          (Receiver &&) r, duration_, loop_};
+    }
+    template <typename Receiver>
+    after_operation<Duration, std::remove_cvref_t<Receiver>> connect(Receiver&& r) const& {
+      return after_operation<Duration, std::remove_cvref_t<Receiver>>{
+          (Receiver &&) r, duration_, loop_};
+    }
```

Comment:
Can we combine these three overloads into a single `const&` overload to avoid duplication here?

Reply:
Can we remove the rvalue overload now?